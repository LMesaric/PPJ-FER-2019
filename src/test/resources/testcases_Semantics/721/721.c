int a[2]={2};
char b[3]="x\"\0";
int main(int a)
{
return a;
}