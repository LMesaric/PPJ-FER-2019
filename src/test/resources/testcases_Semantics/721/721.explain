semantic: too many initializators
note: 'b [ 3 ] = "x\"\0"' on line 2
