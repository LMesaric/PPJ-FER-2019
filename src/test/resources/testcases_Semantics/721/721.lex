KR_INT 1 int
IDN 1 a
L_UGL_ZAGRADA 1 [
BROJ 1 2
D_UGL_ZAGRADA 1 ]
OP_PRIDRUZI 1 =
L_VIT_ZAGRADA 1 {
BROJ 1 2
D_VIT_ZAGRADA 1 }
TOCKAZAREZ 1 ;
KR_CHAR 2 char
IDN 2 b
L_UGL_ZAGRADA 2 [
BROJ 2 3
D_UGL_ZAGRADA 2 ]
OP_PRIDRUZI 2 =
NIZ_ZNAKOVA 2 "x\"\0"
TOCKAZAREZ 2 ;
KR_INT 3 int
IDN 3 main
L_ZAGRADA 3 (
KR_INT 3 int
IDN 3 a
D_ZAGRADA 3 )
L_VIT_ZAGRADA 4 {
KR_RETURN 5 return
IDN 5 a
TOCKAZAREZ 5 ;
D_VIT_ZAGRADA 6 }
