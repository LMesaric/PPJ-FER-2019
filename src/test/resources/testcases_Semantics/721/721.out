<init_deklarator> ::= <izravni_deklarator> OP_PRIDRUZI(2,=) <inicijalizator>
